cmake/developer_package/ncc_naming_style/ncc_naming_style.cmake

```diff
@@ -64,7 +64,11 @@ if(ENABLE_NCC_STYLE)
         find_host_package(Clang QUIET)
     endif()
 
-    if(Clang_FOUND AND TARGET libclang)
+    if(CUSTOM_CLANG_LIB_PATH)
+	    # ncc.py uses Config.set_library_file() so path has to containg libclang.dll
+	    # e.g. for Windows "C:\\Program Files\\LLVM\\bin\\libclang.dll"
+	    set(libclang_location ${CUSTOM_CLANG_LIB_PATH})
```

Comment:
1. please, fix indentation 
2. can we unify / fix cmake script to automatically find `libclang.dll` on Windows without custom paths?

Reply:
1. sure.
2. distribution of llvm for windows does not contain package config file( this is why find_package does not work), but I will try to this in more convenient way.

Reply:
I have improved way of finding Clang on windows.